backend/src/main/java/com/github/backend/service/BoulderService.java

```diff
@@ -1,21 +1,36 @@
 package com.github.backend.service;
 
+import com.github.backend.models.Rating;
 import com.github.backend.repo.BoulderRepo;
 import lombok.RequiredArgsConstructor;
 import com.github.backend.models.Boulder;
 import org.springframework.stereotype.Service;
-
 import java.util.List;
 
+import static java.util.stream.Collectors.toList;
+
 @Service
 @RequiredArgsConstructor
 public class BoulderService {
     private final BoulderRepo boulderRepo;
+    private final UserService userService;
 
     public List<Boulder> getAllBoulders(){
         return boulderRepo.findAll();
     }
     public Boulder getBoulderById(String id) {
         return boulderRepo.findById(id).orElseThrow();
     }
+
+    public Boulder changeRating(String id, double ratingPoints){
+        Rating newRating = new Rating(ratingPoints, userService.getUserById("65f9b9a907ac0162a0072bdc").orElseThrow());
```

Comment:
For the start I'm using a fixed Id of my test-user because I haven't implemented the login yet. Later I will replace it with the UserId of the currently logged-in-user.

Reply:
This is not good practice. You NEVER hard code anything. If you need this later, you will have to implement it in that PR where you really need this.